_people/hyungro_lee.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "Hyungro Lee"
+collection: people
+avatar: hyungro.jpg
```

Comment:
Can you add your picture under `images/people`? Thank you!

Reply:
added 3aa3512bed76db3f6c12283e315626871a95acf9